### PR TITLE
fix: send through app fees when inscripting

### PR DESCRIPTION
**Pull request by victorkirov**
# 🔘 PR Type

What kind of change does this PR introduce?

- [x] Bugfix

# 📜 Background
App fees were not being sent to the apps in the inscription service requests.

Issue Link: #596 

# 🔄 Changes
Enumerate the changes made in this pull request, detailing what has been modified, added, or removed. Include technical details and implications if necessary.

Impact:
- App fees are now sent through with inscription requests

# 🖼 Examples
With 10k app fees:
https://mempool.space/tx/fe304d6f356c13917ab90456beb9aafcf73a12fb23490eaaa2a083e2fcf0e1a6#flow=&vout=0

With no app fees:
https://mempool.space/tx/598f94c6b52007cdd857d70a48034d199960f3c8d95ed5d301d5469ed407724f#flow=&vout=0

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

**Review by DuskaT021 — DISMISSED** [review]

File: src/app/screens/createInscription/index.tsx [hunk 1]
```diff
@@ -254,6 +254,8 @@ function CreateInscription() {
     seedPhrase,
     contentBase64: payloadType === 'BASE_64' ? content : undefined,
     contentString: payloadType === 'PLAIN_TEXT' ? content : undefined,
+    serviceFee: appFee,
```
Comment (JohannesHoppe) [inline src/app/screens/createInscription/index.tsx:257]: I was wondering about this too. When I debugged the extension, I questioned why you don't call it either `serviceFee` or `appFee` in both the extension and sats-connect.

Comment (hans-crypto) [inline src/app/screens/createInscription/index.tsx:257]: I was wondering about this too. When I debugged the extension, I questioned why you don't call it either `serviceFee` or `appFee` in both the extension and sats-connect.

Comment (victorkirov) [reply src/app/screens/createInscription/index.tsx:257]: It was originally called serviceFee and got rename dot appFee in sats-connect. We'll do a renaming in an upcoming PR 👍

Comment (hans-crypto) [reply src/app/screens/createInscription/index.tsx:257]: 👍

**Comment by victorkirov** [comment]
Closing in favor of hot fix branch https://github.com/secretkeylabs/xverse-web-extension/pull/598

**Pull request closed without merging**